src/test/java/org/folio/rest/api/ModesOfIssuanceMigrationScriptTest.java

```diff
@@ -2,24 +2,38 @@
 
 import static org.folio.rest.support.http.InterfaceUrls.modesOfIssuanceUrl;
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
 
 import io.vertx.core.json.JsonObject;
-import org.folio.rest.support.IndividualResource;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
 import java.net.MalformedURLException;
 import java.util.List;
 import java.util.UUID;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeoutException;
+import lombok.SneakyThrows;
+import org.folio.rest.support.IndividualResource;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
 
 public class ModesOfIssuanceMigrationScriptTest extends MigrationTestBase {
   private static final String MIGRATION_SCRIPT = loadScript("renameModesOfIssuance.sql");
 
+  @SneakyThrows
   @Before
   public void beforeEach() {
+    clearData();
+    setupMaterialTypes();
+    setupLoanTypes();
+    setupLocations();
+
+    StorageTestSuite.deleteAll(modesOfIssuanceUrl(""));
+
+    removeAllEvents();
+  }
+
+  @After
+  public void afterEach() {
     StorageTestSuite.deleteAll(modesOfIssuanceUrl(""));
   }
```

Comment:
Why is this now being done before and after each test?

Reply:
Cool, thanks. FWIW I prefer before rather than after, because the responsibility for a clean slate lives with the tests that want it